libs/orbit-essentials/src/repository.rs

```diff
@@ -25,6 +25,11 @@ where
     /// Adds the indexes for the current value.
     fn add_entry_indexes(&self, value: &Value);
 
+    /// Clears all the indexes.
+    fn clear_indexes(&self) {
```

Comment:
should this default implementation exist? one might forget to implement this for a repository and rust wont complain.